src/adapter/src/catalog/builtin.rs

```diff
@@ -1790,6 +1790,22 @@ pub static MZ_STORAGE_HOST_METRICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSo
     is_retained_metrics_relation: true,
 });
 
+// This will be replaced with per-replica tables with source/sink multiplexing is supported.
+pub static MZ_STORAGE_SOURCE_STATISTICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
+    name: "mz_storage_source_statistics",
+    schema: MZ_INTERNAL_SCHEMA,
+    data_source: Some(IntrospectionType::StorageSourceStatistics),
+    desc: RelationDesc::empty()
+        .with_column("id", ScalarType::String.nullable(false))
+        .with_column("worker_id", ScalarType::UInt64.nullable(false))
+        // These are nullable as we may not fill them all out at the same time.
```

Comment:
I'm not seeing where this happens—and even if it does, can't we just default them to `false`/`0`?

Reply:
this was from an earlier version, so i can just use `.nullable(true)` here, because they are always set